packages/common/src/types/projects.ts

```diff
@@ -153,6 +153,7 @@ export type CreateSnowflakeCredentials = {
     accessUrl?: string;
     startOfWeek?: WeekDay | null;
     quotedIdentifiersIgnoreCase?: boolean;
+    override?: string;
```

Comment:
we should run a `generate-api`

Reply:
I already did, I believe this endpoint is old and not using tsoa, so it doesn't affect the docs.

Reply:
Interesting, I got a change in `routes`

Reply:
ups, you were right, sorry. I don't know what happened, I was probably in a different branch. 

Just pushed